documentation/get-started/tutorials/get-started-qfs.en.md

```diff
@@ -116,6 +116,9 @@ tile size is set to 1, each raster pixel will have a spatial extent of
 1x1 m, if it is set to 1000, each raster pixel will have a spatial
 extent of 1 square kilometer.
 
+!!! note
+   Base map generation is disabled on QFieldCloud.
```

Comment:
```suggestion
   Base map generation is disabled on QFieldCloud. You can still add your basemap by manually exporting it into `mbtiles` or `tif` file in advance.
```

Reply:
@suricactus , @SeqLaz  , could be good to explicitly refer to the processing algorithm that allows user to do that, i.e. Generate XYZ tiles (MBTiles)

Reply:
@nirvn Updated my comment above. Was thinking about the same but I was too lazy to check the names. Feel free to approve so we can have this on the docs site :)

Reply:
Initially I was doing a whole documentation of that!